apps/ledger-live-desktop/tests/specs/speculos/accountSpeculos.spec.ts

```diff
@@ -0,0 +1,89 @@
+import test from "../../fixtures/common";
+import { expect } from "@playwright/test";
+import { PortfolioPage } from "../../models/PortfolioPage";
+import { AddAccountModal } from "../../models/AddAccountModal";
+import { Layout } from "../../models/Layout";
+import { AccountPage } from "../../models/AccountPage";
+import { AccountsPage } from "../../models/AccountsPage";
+import { specs } from "../../utils/speculos";
+import { Currency } from "../../enum/Currency";
+
+const currencies: Currency[] = [
+  Currency.BTC,
+  Currency.tBTC,
+  Currency.ETH,
+  Currency.tETH,
+  Currency.XRP,
+  Currency.SOL,
+  Currency.DOT,
+  Currency.TRX,
+];
+
+for (const currency of currencies) {
+  test.describe.parallel("Accounts @smoke", () => {
+    test.use({
+      userdata: "skip-onboarding",
+      testName: `receiveSpeculos_${currency.uiName}`,
+      speculosCurrency: specs[currency.deviceLabel.replace(/ /g, "_")],
+      speculosOffset: Math.floor(Math.random() * 10000),
+    });
+    let firstAccountName = "NO ACCOUNT NAME YET";
+    let accountsListBeforeRemove: (string | null)[];
+    test(`[${currency.uiName}] Add account`, async ({ page }) => {
```

Comment:
could you please add as a comment above this line which test case IDs correspond to this automated test from xray? Same for every test case in speculos folder